website/src/pages/docs/features/subscriptions.mdx

```diff
@@ -365,7 +365,47 @@ const executeSubscription = (
 const network = Network.create(executeQueryOrMutation, executeSubscription)
 ```
 
-## Subscriptions over WebSockets with `graphql-ws`
+## GraphQL over Server-Sent Events Protocol (via `graphql-sse`)
+
+In case you want the subscriptions to be transported following the [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md), you simply use the `@graphql-yoga/plugin-graphql-sse` plugin for GraphQL Yoga that exposes an additional endpoint (defaulting to `/graphql/stream`) used for [graphql-sse](https://github.com/enisdenjo/graphql-sse) clients.
```

Comment:
It would be great to have a comparison on what is supported out of the box without the plugin and what requires the plugin.

Reply:
Yoga is not spec compliant at all, not even with the "distinct connection mode".

Reply:
Then let's document that and why you would choose one over the other.
Otherwise people will be confused on what to use.